### ISSUE TITLE
Cannot build because invalid JVM target

### ISSUE THREAD

**Issue by hanssonduck** [rickclephas/KMM-ViewModel#44]
I'm using the latest Android Studio Canary and KMM/KMP plugins.
This won't compile and throws the following error.

```kotlin
class TestViewModel : KMMViewModel() {
    @NativeCoroutinesState
    var test = MutableStateFlow(viewModelScope, value = "Hello World!")
}
```

```bash
:shared:compileDebugKotlinAndroid
./TestViewModel.kt:9:16 Cannot inline bytecode built with JVM target 11 into bytecode that is being built with JVM target 1.8. Please specify proper '-jvm-target' option
```

I've tried to search for a solution and tried everything but nothing works.

**Comment by hanssonduck** [comment]
Got it working, Gradle is so fun sometimes.

**Comment by lmiskovic** [comment]
Hey, just to provide some more info regarding this issue although I see it is closed.
I've just encountered it and it was resolved by suppressing compiler warning. I've added `@file:Suppress("INLINE_FROM_HIGHER_PLATFORM")` to affected file and it is compiling again.
Credits goes to: 
https://youtrack.jetbrains.com/issue/KTIJ-20816/Bogus-error-Cannot-inline-bytecode-built-with-JVM-target-11-into-bytecode-that-is-being-built-with-JVM-target-1.8.#focus=Comments-27-7378342.0-0
